src/components/Home/CardInfo/styles.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  background-color: #fff;
+  border-radius: 0.4rem;
+  overflow: hidden;
+  box-shadow: 0 3rem 6rem rgba(0, 0, 0, 0.1);
+  cursor: pointer;
+  transition: 0.2s;
+
+  &:hover {
+    transform: translateY(-0.5%);
+    box-shadow: 0 4rem 8rem rgba(0, 0, 0, 0.5);
+  }
+`;
+
+export const Card = styled.div``;
+
+export const Content = styled.div`
+  padding: 3rem;
+`;
+
+export const Title = styled.h1`
+  font-size: 3rem;
+  font-weight: 500;
+  color: #0d0d0d;
+  margin-bottom: 1.5rem;
+`;
+
+export const Description = styled.p`
+  font-size: 1.6rem;
+  letter-spacing: 0.1rem;
+  line-height: 1.7;
+  color: #3d3d3d;
+  margin-bottom: 2.5rem;
+`;
+
+export const ViewMore = styled.button`
+display:block;
```

Comment:
This should have triggered a lint error. Try using your text editor capabilities to help you with linting. If you are using vscode here's a useful article about it: https://www.digitalocean.com/community/tutorials/linting-and-formatting-with-eslint-in-vs-code